app/src/Controller/ApiController.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use App\Api\GameApi;
+use App\Api\TokenGenerator;
+use App\Entity\Game;
+use App\Entity\Player;
+use App\Repository\GameRepository;
+use App\Repository\PlayerRepository;
+use App\ArgumentResolver\GameContext;
+
+class ApiController extends Controller {
+  public const DEFAULT_SIZE = 4;
+
+  private $serializer;
+  private $api;
+  private $gameRepository;
+  private $playerRepository;
+
+  public function __construct(GameApi $gameApi, GameRepository $gameRepository, PlayerRepository $playerRepository) {
+    $this->api = $gameApi;
+    $this->gameRepository = $gameRepository;
+    $this->playerRepository = $playerRepository;
+  }
+
+  public function new(Request $request) {
+    $mode = $request->get('mode');
+    $apiResponse = $this->api->new(self::DEFAULT_SIZE, $mode && $mode == 'multi');
```

Comment:
Prefer strict equality check ===

Reply:
$mode is not used elsewhere. You can directly do $request->get('mode') === 'multi'